connect/runtime/src/test/java/org/apache/kafka/connect/integration/ConnectorValidationIntegrationTest.java

```diff
@@ -331,7 +334,7 @@ public void testConnectorHasAbstractConverter() throws InterruptedException {
     @Test
     public void testConnectorHasConverterWithNoSuitableConstructor() throws InterruptedException {
         Map<String, String> config = defaultSinkConnectorProps();
-        config.put(KEY_CONVERTER_CLASS_CONFIG, TestConverterWithPrivateConstructor.class.getName());
+        config.put(KEY_CONVERTER_CLASS_CONFIG, TestPlugins.TestPlugin.ALWAYS_THROW_EXCEPTION.className());
```

Comment:
I think we should use `BAD_PACKAGING_DEFAULT_CONSTRUCTOR_PRIVATE_CONNECTOR` in this case as well?

Reply:
oops, I misread the code, now I've corrected it.